js/ui/menu/ui.menu.js

```diff
@@ -373,7 +373,7 @@ var Menu = MenuBase.inherit({
     },
 
     _toggleAdaptiveMode: function(state) {
-        var $menuItemsContainer = this.element().find("." + DX_MENU_ITEMS_CONTAINER_CLASS),
+        var $menuItemsContainer = this.element().find("." + DX_MENU_ITEMS_CONTAINER_CLASS).parent(),
```

Comment:
I suppose you should save "menuItemsContainer" to this._itemsContainer on creating.